### PR TITLE
Ratvarian Spears are now sharp

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
add: Ratvarian Spears are now sharp and can be used to cut off limbs or butcher mobs.
/:cl:

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Isn't sharp accurate supposed to be surgical tools?

**Comment by ChangelingRain** [comment]
it's like sharp but more sharper

**Comment by KorPhaeron** [comment]
have you every done heart surgery with a steampunk spear

**Comment by Zilenan91** [comment]
Why can spears cut off limbs anyways, that's not really what spears do.

**Comment by ChangelingRain** [comment]
all IS_SHARP_ACCURATE actually does is let you cut slice-able food accurately.

also this is clearly a bladed spear not just a pointy spear duh

File: code/game/gamemodes/clock_cult/clock_items.dm [hunk 1]
```diff
@@ -702,8 +702,9 @@
 	icon = 'icons/obj/clockwork_objects.dmi'
 	icon_state = "ratvarian_spear"
 	item_state = "ratvarian_spear"
-	force = 17 //Extra damage is dealt to silicons in afterattack()
+	force = 17 //Extra damage is dealt to silicons in attack()
 	throwforce = 40
+	sharpness = IS_SHARP_ACCURATE
```
Comment (xxalpha) [inline code/game/gamemodes/clock_cult/clock_items.dm]: >accurate

hmmm

**Comment by Zilenan91** [comment]
That's more of a pike at that point if a spear is used to slice rather than stab

**Comment by Supermichael777** [comment]
spears were used to slash. you only poked in large formations or specific hunting situations(a little nick ain't gonna do shit to a boar). A blade that wide would almost certainly necessitate  slashing. Think of a spear as knife on a stick. The crappy crafted spears are like shivs, all you can to is stab.

**Pull request merged**